crates/storage/src/traits/impls/prims.rs

```diff
@@ -288,6 +288,53 @@ where
     }
 }
 
+impl SpreadLayout for AccountId
+where
+    Self: PackedLayout,
+{
+    const FOOTPRINT: u64 = 1_u64;
+    const REQUIRES_DEEP_CLEAN_UP: bool = false;
+
+    fn pull_spread(ptr: &mut KeyPtr) -> Self {
+        crate::traits::impls::forward_pull_packed::<Self>(ptr)
+    }
+
+    fn push_spread(&self, ptr: &mut KeyPtr) {
+        crate::traits::impls::forward_push_packed::<Self>(self, ptr)
+    }
+
+    fn clear_spread(&self, ptr: &mut KeyPtr) {
+        crate::traits::impls::forward_clear_packed::<Self>(self, ptr)
+    }
+}
+
+// TODO: Not sure if this can actually be implemented, since we need some sort of "sane" default
```

Comment:
Basically just blocked on this. I think we'll be able to sidestep this with the new storage re-write in #1217.